probe-rs/src/probe/ftdi/mod.rs

```diff
@@ -354,9 +353,9 @@ impl JtagAdapter {
         self.shift_ir(&ir.to_le_bytes(), irbits)?;
 
         let drbits = params.drpre + len_bits + params.drpost;
-        let request = if let Some(data) = data {
-            let mut data = BitVec::<Lsb0, u8>::from_slice(data);
-            data.truncate(len_bits);
+        let request = if let Some(data_slice) = data {
+            let data = BitSlice::<Lsb0, u8>::from_slice(data_slice).unwrap();
```

Comment:
Please return an error here, instead of unwrapping.

Reply:
I thought this is unlikely (https://docs.rs/bitvec/0.19.4/src/bitvec/slice.rs.html#525) but I changed it cause I need to learn more about proper error handling anyway